README.md

```diff
@@ -1 +1,40 @@
-Refer to respective folders for assignments
\ No newline at end of file
+# Data Engineering challenge assignment for Wealth42
+
+Please follow the instructions in order to run the program successfully.
+
+Pre-requisite: Python and MYSQL server should be installed.
+
+## INSTRUCTIONS
+
+1. Install all the required packages by typing the following command in your command prompt
+	
+	<pre>pip install -r requirements.txt</pre>
+
+2. Now store your host, username and password of your mysql server in environment variables as Host, user and Password respectively.
+
+3. Now to create database "bikepoint" and then table "bikepoints data" inside that database run the file <strong><u>create_database.py</u></strong> by command
+
+	<pre>python create_database.py</pre>
+
+4. Next, to insert/update data into your database table "bikepoints data" run the file <strong><u>Extract&Load.py</u></strong> by command 
+	
+	<pre>python Extract&Load.py</pre>
+
+5. Now to update data every 5 minutes, we need to schedule the file <strong><u>Extract&Load.py</u></strong> to run every 5 minutes for that follow the steps below:

```

Comment:
Can you also include instructions using crontab?

Reply:
okay, I'll add that too.